pkg/systemd/services/services-list.jsx

```diff
@@ -35,17 +35,22 @@ export const ServicesList = ({ units, isTimer }) => {
                   id="services-list"
                   onSelectDataListItem={id => cockpit.location.go([id])}
                   className="services-list">
-            { units.map(unit => <ServicesRow key={unit.path} isTimer={isTimer} {...unit} />) }
+            { units.map(unit => <ServicesRow key={unit[0]} isTimer={isTimer} shortId={unit[0]} {...unit[1]} />) }
```

Comment:
Before this PR units property was a list of Objects, like [{ Id: 'test 0 Id', Description: 'test 0' }, { Id: 'test 1 Id', 'Description': 'test 1' }]
These unit objects come from the Services parent component's react state, which has an object called `units_by_path` with keys the unit DBus object path and values these unit object from above.

Previously we were using Array.map function in order to get from the `units_by_path` the values which we were passing up to the ServicesList component. That approach though was hiding the units which are symlinks since they were pointing to the same DBus object path, so only one of a group of symlinked units would appear in the list.

With this commit I changed how I contruct the units property of the ServicesList, so that the duplicate units would appear in the list as well.
https://github.com/cockpit-project/cockpit/pull/13984/files#diff-f308bf5d83fe34c1e4da8140ee4766dbL720

As seen in the above diff I now have the `path_by_id` which is a mapping of the seen units Ids to units DBus Object paths. This contains the symlinks as well.
And here https://github.com/cockpit-project/cockpit/pull/13984/files#diff-f308bf5d83fe34c1e4da8140ee4766dbR728
I convert it from list of Ids to list of objects. shortId is the name of the unit that we display in the UI which might be also symlink.
A cleaner version of the code would be to construct an array like:
```
const units = Object.keys(this.path_by_id)
...
    .map(unit_id => Object.assign({ shortId: unit_id }, unit_by_path[this.path_by_id[unit_id]]]);
````

But this makes the whole unit object to be copied over, which makes the performance of render really slower.
So to improve performance I instead create this two elements array, containing the shortId and the reference to the unit object, which will be reused for all symlinked units.